opentripplanner-geocoder/pom.xml

```diff
@@ -31,6 +31,18 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
   </properties>
 
+<!-- CUTR SNAPSHOTs/RELEASES -->

```

Comment:
You can actually go ahead and remove this `<repositories>` block - I finally got the v1.0.0 artifacts published to Maven Central last night, so we can use those.